src/client/src/ui/blotter/components/BlotterHeader.tsx

```diff
@@ -0,0 +1,96 @@
+import { GridApi } from 'ag-grid'
+import React, { Component } from 'react'
+import { styled } from 'rt-util'
+import { columnDefinitions } from './blotterUtils'
+import BlotterToolbar from './toolbar/BlotterToolbar'
+
+interface BlotterHeaderProps {
```

Comment:
just opinion but since these interfaces are only used for the one component they could be called 'Props' and 'State'